src/porepy/models/contact_mechanics_model.py

```diff
@@ -430,15 +430,13 @@ def newton_iteration(assembler, setup, u0, tol=1e-14, solver=None):
     A, b = assembler.assemble_matrix_rhs()
     print("max A: {0:.2e}".format(np.max(np.abs(A))))
     print(
-        "max: {0:.2e} and min: {1:.2e} A sum: ".format(
+        "max: {0:.2e} and min: {1:.2e} A sum.".format(
```

Comment:
I think you can remove these print statements, or maybe better change then to  logger.debug(".......")

Reply:
Good idea!